horace_core/sqw/@sqw/sqw.m

```diff
@@ -88,7 +88,11 @@
         wout = noisify(w,varargin);
         
         function dtp = my_detpar(obj)
-            dtp = obj.detpar_x;
+            if ~isempty(obj.detpar_x)
```

Comment:
If both are empty this will error. Expected?

Reply:
obj in this case is a sqw (or array thereof). Guess it is conceivable that obj could be an empty array with the right type. If it errors maybe we rewrite in python?